app/src/main/java/com/checkin/app/checkin/session/SessionRepository.kt

```diff
@@ -186,5 +188,27 @@ class SessionRepository private constructor(context: Context) : BaseRepository()
         }.asLiveData
     }
 
+    val customerClosedSessionList: LiveData<Resource<List<ClosedSessionBriefModel>>> =
```

Comment:
This still has the same issue I mentioned here:
https://github.com/checkin-team/Checkin-Android/pull/83#discussion_r392626193
Plz fix it ASAP. It's a really big bug. You haven't tested and see that your API call only happens once, never after that.